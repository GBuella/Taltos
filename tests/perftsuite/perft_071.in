setboard K7/p7/k7/8/8/8/8/8 b - - 0 1
echo K7/p7/k7/8/8/8/8/8 b - - 0 1
qperft 1
qperft 2
qperft 3
qperft 4
qperft 5
qperft 6
