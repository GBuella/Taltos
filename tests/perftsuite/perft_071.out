K7/p7/k7/8/8/8/8/8 b - - 0 1
3
7
43
199
1347
6249
